assets/sass/application-ie8.sass

```diff
@@ -4,8 +4,15 @@ $path: "/assets/images/"
 @import 'govuk/all-ie8'
 @import 'moj/all'
 
-@import './components/header-bar'
-@import './components/card'
+@import "./constants/colours"
+
 @import './components/add-another'
+@import './components/card'
```

Comment:
I doubt that this will fix the IE backlink issue. `js-backlink` class is does not have any CSS, its just used as an identifier in JS

Reply:
I would maybe suggest trying this in `govukFrontendInit.js`? looks like `Jquery.click()` is deprecated

```
// Initiate the back links
$('[class$=js-backlink]').on('click', e => {
  e.preventDefault()
  if ($('ul.govuk-error-summary__list').length > 0) {
    window.history.go(-2)
  } else {
    window.history.go(-1)
  }
})
```

Reply:
Yeah I did look at this JS as well, though this particular change was just to update the ie8 module to include all the same SCCS modules as we include in application.sccs. I'll alter the JS above to see if that deprecated click() method is the issue for IE.